README.md

```diff
@@ -6,39 +6,76 @@ The `irb` command from your shell will start the interpreter.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+> **Note**
+>
+> If you use Ruby 2.4 or later versions, a bundled version of IRB should already be installed with it by default.
```

Comment:
What does this mean? I'm not sure if I said this in that article. IRB has been a standard library in versions older than 2.4 (`binding.irb` is just a new feature added in that version), and it's converted to a (default) gem in Ruby 2.6.

Reply:
In other words, `gem install irb` or `gem 'irb'` works only for Ruby 2.6+.

Reply:
Ah sorry I misread the article. What I want to express is that:

1. Each version of Ruby comes with a certain version of IRB
2. But users can chose to upgrade the version using the installation instructions